### PR TITLE
Create amazon advertising destination

### PR DESCRIPTION
**Pull request by vinvishy**

### CODE REVIEW

File: help/destinations/catalog/advertising/amazon-advertising.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+Amazon Ads offers a range of options to help you achieve your advertising goals to registered sellers, vendors, book vendors, Kindle Direct Publishing (KDP) authors, app developers, and/or agencies.
+
+The Amazon Ads integration with Adobe Experience Platform provides turn-key integration to Amazon Ads products, including the Amazon DSP (ADSP). Using the Amazon Ads destination in Adobe Experience Platform, users are able to define advertiser audiences for targeting and activation on the Amazon DSP.
+
+
+## Use cases
+
+To help you better understand how and when you should use the Amazon Advertising destination, here are sample use cases that Adobe Experience Platform customers can solve by using this destination.
+
+**For Activation and Targeting Use-cases:**
```
Comment (vfalon) [inline help/destinations/catalog/advertising/amazon-advertising.md]: Line 14: Please change "For Activation and Targeting Use-cases" to just "Activation and targeting".

File: help/destinations/catalog/advertising/amazon-advertising.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+This integration with Amazon DSP allows Amazon Ads advertisers to pass advertiser CDP segments from Adobe Experience Platform to Amazon’s DSP to create advertiser audiences for advertising targeting. Audiences may be selected within the Amazon DSP for positive targeting, as well as negative targeting (suppression). Additionally, using signals generated through Amazon Marketing Cloud, advertisers can optimize their advertiser audiences which will synchronize audience changes with Amazon DSP.
+
+
+## Prerequisites
+
+*In order to use the Amazon Advertising connection with Adobe Experience Platform, users must first have access to an Amazon DSP Advertiser Account.  To provision these instances, please visit the following page on the Amazon Advertising website:*
```
Comment (vfalon) [inline help/destinations/catalog/advertising/amazon-advertising.md]: Please remove the italicized formatting in the prerequisites section (the link too)

File: help/destinations/catalog/advertising/amazon-advertising.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+* *[Get started with Amazon DSP](https://advertising.amazon.com/solutions/products/amazon-dsp?ref_=a20m_us_hnav_p_dsp_adtech)*
+
+## Supported identities
+
+Amazon Advertising connection supports the activation of identities described in the table below. Learn more about [identities](https://experienceleague.adobe.com/docs/experience-platform/identity/namespaces.html?lang=en). For more details on Amazon Advertising’s supported identities, visit this [page](https://advertising.amazon.com/dsp/help/ss/en/audiences#GA6BC9BW52YFXBNE).
```
Comment (vfalon) [inline help/destinations/catalog/advertising/amazon-advertising.md]: Line 26: "The Amazon Advertising connection[...]"

File: help/destinations/catalog/advertising/amazon-advertising.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+This integration with Amazon DSP allows Amazon Ads advertisers to pass advertiser CDP segments from Adobe Experience Platform to Amazon’s DSP to create advertiser audiences for advertising targeting. Audiences may be selected within the Amazon DSP for positive targeting, as well as negative targeting (suppression). Additionally, using signals generated through Amazon Marketing Cloud, advertisers can optimize their advertiser audiences which will synchronize audience changes with Amazon DSP.
+
+
+## Prerequisites
+
+*In order to use the Amazon Advertising connection with Adobe Experience Platform, users must first have access to an Amazon DSP Advertiser Account.  To provision these instances, please visit the following page on the Amazon Advertising website:*
+
+* *[Get started with Amazon DSP](https://advertising.amazon.com/solutions/products/amazon-dsp?ref_=a20m_us_hnav_p_dsp_adtech)*
+
+## Supported identities
```
Comment (vfalon) [inline help/destinations/catalog/advertising/amazon-advertising.md]: Let's rephrase "For more details on Amazon Advertising’s supported identities, visit[...]" to "For more details on the identities supported by Amazon Advertising, visit [...]"

File: help/destinations/catalog/advertising/amazon-advertising.md (hunk 3 above)
Comment (vfalon) [inline help/destinations/catalog/advertising/amazon-advertising.md]: This link asks me to log in to an Amazon account. Is there a public documentation link that you could link instead?

File: help/destinations/catalog/advertising/amazon-advertising.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+
+Amazon Advertising connection supports the activation of identities described in the table below. Learn more about [identities](https://experienceleague.adobe.com/docs/experience-platform/identity/namespaces.html?lang=en). For more details on Amazon Advertising’s supported identities, visit this [page](https://advertising.amazon.com/dsp/help/ss/en/audiences#GA6BC9BW52YFXBNE).
+
+|Target Identity	|Description	|Considerations	|
+|---	|---	|---	|
+|phone_sha256	|Phone numbers hashed with the SHA256 algorithm	|Both plain text and SHA256 hashed phone numbers are supported by Adobe Experience Platform. When your source field contains unhashed attributes, check the **Apply transformation** option, to have Platform automatically hash the data on activation.	|
```
Comment (vfalon) [inline help/destinations/catalog/advertising/amazon-advertising.md]: Please keep the description formatting in the identities table as provided in the [template](https://experienceleague.adobe.com/docs/experience-platform/destinations/destination-sdk/document-destination/self-service-template.html?lang=en). The UICONTROL and DNL tags are required for correct localization.

Dropping below the table from the template:

|phone_sha256|Phone numbers hashed with the SHA256 algorithm|Both plain text and SHA256 hashed phone numbers are supported by Adobe Experience Platform. When your source field contains unhashed attributes, check the **[!UICONTROL Apply transformation]** option, to have [!DNL Platform] automatically hash the data on activation.|
|email_lc_sha256|Email addresses hashed with the SHA256 algorithm|Both plain text and SHA256 hashed email addresses are supported by Adobe Experience Platform. When your source field contains unhashed attributes, check the **[!UICONTROL Apply transformation]** option, to have [!DNL Platform] automatically hash the data on activation.|

File: help/destinations/catalog/advertising/amazon-advertising.md [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+|email_lc_sha256	|Email addresses hashed with the SHA256 algorithm	|Both plain text and SHA256 hashed email addresses are supported by Adobe Experience Platform. When your source field contains unhashed attributes, check the **Apply transformation** option, to have Platform automatically hash the data on activation.	|
+
+## Export type and frequency
+
+
+Refer to the table below for information about the destination export type and frequency.
+
+|Item	|Type	|Notes	|
```
Comment (vfalon) [inline help/destinations/catalog/advertising/amazon-advertising.md]: Please keep the formatting in the export type and frequency table as provided in the [template](https://experienceleague.adobe.com/docs/experience-platform/destinations/destination-sdk/document-destination/self-service-template.html?lang=en). The UICONTROL and DNL tags are required for correct localization.

File: help/destinations/catalog/advertising/amazon-advertising.md [hunk 7]
```diff
@@ -0,0 +1,112 @@
+# Amazon Ads connection
```
Comment (vfalon) [inline help/destinations/catalog/advertising/amazon-advertising.md]: What is the actual name of this connection? Is it Amazon Ads or Amazon Advertising? I see them being used interchangeably in the document. Please use the naming that will be displayed in the Platform UI.

Comment (vfalon) [inline help/destinations/catalog/advertising/amazon-advertising.md]: There are multiple issues with how this document was authored. It looks like it was authored based on the [rendered HTML template](https://experienceleague.adobe.com/docs/experience-platform/destinations/destination-sdk/document-destination/self-service-template.html?lang=en), which is not the correct workflow.

Please re-author the documentation starting from the [Markdown template](https://experienceleague.adobe.com/docs/experience-platform/assets/yourdestination-template.zip?lang=en), and keep all the formatting in place.

- The document seems to be missing all the Adobe formatting that is provided in the self-service documentation [template](https://experienceleague.adobe.com/docs/experience-platform/destinations/destination-sdk/document-destination/self-service-template.html?lang=en). The formatting and Adobe-specific tags are required for correct localization and deep linking.
Please review the documentation and keep the formatting in place.
- UICONTROL and DNL tags, heading anchor tags, all of these are missing from the submitted document. Please review the entire document and add them back. 
- The self-service template includes relative links to other sections of our documentation, linking directly to the Markdown files. In your submission, the links have been changed to public HTML links on Experience League.

**Review by vfalon — CHANGES_REQUESTED** [review]
Hi @vinvishy ! Thank you for submitting the documentation. However, there are multiple issues with how this document was authored. It looks like it was authored based on the [rendered HTML template](https://experienceleague.adobe.com/docs/experience-platform/destinations/destination-sdk/document-destination/self-service-template.html?lang=en), which is not the correct workflow.

- The document is missing all the Adobe formatting that is provided in the self-service documentation [template](https://experienceleague.adobe.com/docs/experience-platform/destinations/destination-sdk/document-destination/self-service-template.html?lang=en). The formatting and Adobe-specific tags are required for correct localization and deep linking.
- UICONTROL and DNL tags, heading anchor tags, all of these are missing from the submitted document. Please review the entire document and add them back. 
- The self-service template includes relative links to other sections of our documentation, linking directly to the Markdown files. In your submission, the links have been changed to public HTML links on Experience League.

The easiest way is to re-author the documentation correctly, starting from the [Markdown template](https://experienceleague.adobe.com/docs/experience-platform/assets/yourdestination-template.zip?lang=en) (download link), and keep all the formatting in place. Do not change any links or formatting.

Once we receive the new submission, we'll review the content and go from there.
Thanks!

File: help/destinations/catalog/advertising/amazon-advertising.md (hunk 1 above)
Comment (vinvishy) [reply help/destinations/catalog/advertising/amazon-advertising.md]: Done.

File: help/destinations/catalog/advertising/amazon-advertising.md (hunk 2 above)
Comment (vinvishy) [reply help/destinations/catalog/advertising/amazon-advertising.md]: Done.

File: help/destinations/catalog/advertising/amazon-advertising.md (hunk 3 above)
Comment (vinvishy) [reply help/destinations/catalog/advertising/amazon-advertising.md]: Done.

File: help/destinations/catalog/advertising/amazon-advertising.md (hunk 4 above)
Comment (vinvishy) [reply help/destinations/catalog/advertising/amazon-advertising.md]: Done.

File: help/destinations/catalog/advertising/amazon-advertising.md (hunk 3 above)
Comment (vinvishy) [reply help/destinations/catalog/advertising/amazon-advertising.md]: We do not have a public facing documentation link. We do not share our hashing logic to non-customers.

File: help/destinations/catalog/advertising/amazon-advertising.md (hunk 5 above)
Comment (vinvishy) [reply help/destinations/catalog/advertising/amazon-advertising.md]: Done.

File: help/destinations/catalog/advertising/amazon-advertising.md (hunk 6 above)
Comment (vinvishy) [reply help/destinations/catalog/advertising/amazon-advertising.md]: Done.

File: help/destinations/catalog/advertising/amazon-advertising.md (hunk 7 above)
Comment (vinvishy) [reply help/destinations/catalog/advertising/amazon-advertising.md]: Done.

Comment (vinvishy) [reply help/destinations/catalog/advertising/amazon-advertising.md]: We've converted everything to "Amazon Ads" now. We have also updated the platform UI to say Amazon Ads.

File: help/destinations/catalog/advertising/amazon-ads.md [hunk 8]
```diff
@@ -0,0 +1,133 @@
...
+
+>[!IMPORTANT]
+>
+>This documentation page was created by the *Amazon Ads* team. For any inquiries or update requests, please contact them directly at *`amc-support@amazon.com`.*
+
+## Use cases {#use-cases}
+
+To help you better understand how and when you should use the *Amazon Ads* destination, here are sample use cases that Adobe Experience Platform customers can solve by using this destination.
+
+### Use case #1 {#use-case-1}
```
Comment (vfalon) [inline help/destinations/catalog/advertising/amazon-ads.md]: If this is the only use case in the doc, please:

- remove `### Use case #1 {#use-case-1}` 
- Update `Activation and Targeting Use-cases:` to `### Activation and targeting {#activation-and-targeting}`

File: help/destinations/catalog/advertising/amazon-ads.md [hunk 9]
```diff
@@ -0,0 +1,133 @@
...
+
+To connect to this destination, follow the steps described in the [destination configuration tutorial](../../ui/connect-destination.md). In the configure destination workflow, fill in the fields listed in the two sections below.
+
+### Authenticate to destination {#authenticate}
+
+To authenticate to the destination, fill in the required fields and select **[!UICONTROL Connect to destination]**.
+
+To connect to the Amazon Ads destination, you will first click to establish a connection via OAuth2. After clicking *Connect to destination*, you will be taken to the Amazon Ads connection interface where you will first select the advertiser accounts you wish to connect to.  Upon connection, you will be redirected back to Adobe Experience Platform with a new connection, provided with the ID of the Advertiser Account you selected. Select the appropriate Advertiser Account on the destination configuration screen to proceed.
```
Comment (vfalon) [inline help/destinations/catalog/advertising/amazon-ads.md]: Line 69: The first phrase is a bit redundant, considering the one before it (which is part of the template). Let's remove the part up to " After clicking *Connect to destination*,", and just start with "You will be taken to the Amazon Ads[...]"

File: help/destinations/catalog/advertising/amazon-ads.md [hunk 10]
```diff
@@ -0,0 +1,133 @@
...
+
+
```
Comment (vfalon) [inline help/destinations/catalog/advertising/amazon-ads.md]: Any specific reason for the extra blank lines? Are you planning to add more content?

File: help/destinations/catalog/advertising/amazon-ads.md [hunk 11]
```diff
@@ -0,0 +1,133 @@
...
+
+* **[!UICONTROL Bearer token]**: Fill in the bearer token to authenticate to the destination.
+
+### Fill in destination details {#destination-details}
+
+To configure details for the destination, fill in the required and optional fields below. An asterisk next to a field in the UI indicates that the field is required.
+
+
```
Comment (vfalon) [inline help/destinations/catalog/advertising/amazon-ads.md]: Any specific reason for the extra blank lines? Are you planning to add more content?

File: help/destinations/catalog/advertising/amazon-ads.md [hunk 12]
```diff
@@ -0,0 +1,133 @@
...
+
+*  **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
+*  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+*  **[!UICONTROL Amazon Ads Advertiser Id]**: Select the ID for the target Amazon Ads account used for the destination.
```
Comment (vfalon) [inline help/destinations/catalog/advertising/amazon-ads.md]: Please replace [!UICONTROL Amazon Ads Advertiser Id] with [!UICONTROL Amazon Ads Advertiser ID]. Also, make sure to capitalize ID in the labels configured in your destination submission, too.

File: help/destinations/catalog/advertising/amazon-ads.md [hunk 13]
```diff
@@ -0,0 +1,133 @@
...
+
+Note: once selecting this Amazon Ads Advertiser Id, you will need to create a new destination to change this. If you re-authenticate the OAuth credentials and select a new Advertiser Id, your changes will not apply.
```
Comment (vfalon) [inline help/destinations/catalog/advertising/amazon-ads.md]: Capitalize ID.

File: help/destinations/catalog/advertising/amazon-ads.md [hunk 14]
```diff
@@ -0,0 +1,133 @@
...
+
+Read [Activate profiles and segments to streaming segment export destinations](/help/destinations/ui/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination.
+
+### Map attributes and identities {#map}
+
+The Amazon Ads connection supports hashed email address and hashed phone numbers for identity matching purposes.  The screenshot below provides an example matching that is compatible with the Amazon Ads connection:
+
+![Adobe to Amazon Ads mapping](../../assets/catalog/advertising/amazon_ads_image_2.png)
+
+* When mapping email, select the Lower case, SHA256 source field (Email_LC_SHA256)
```
Comment (vfalon) [inline help/destinations/catalog/advertising/amazon-ads.md]: To map hashed email addresses, select the `Email_LC_SHA256` identity namespace as a source field.

File: help/destinations/catalog/advertising/amazon-ads.md [hunk 15]
```diff
@@ -0,0 +1,133 @@
...
+* When mapping phone, select the SHA256 source field (Phone_SHA256)
```
Comment (vfalon) [inline help/destinations/catalog/advertising/amazon-ads.md]: To map hashed phone numbers, select the `Phone_SHA256` identity namespace as a source field.

File: help/destinations/catalog/advertising/amazon-ads.md [hunk 16]
```diff
@@ -0,0 +1,133 @@
...
+* You may also select fields containing email address or phone numbers to map to the corresponding target field in the Amazon Ads connection; in this case, you must select `Apply Transformation`
```
Comment (vfalon) [inline help/destinations/catalog/advertising/amazon-ads.md]: To map unhashed email addresses or phone numbers, select the corresponding identity namespaces as source fields, and check the `Apply Transformation` option to have Platform hash the identities on activation.

File: help/destinations/catalog/advertising/amazon-ads.md [hunk 17]
```diff
@@ -0,0 +1,133 @@
...
+
+It is strongly recommended that you map as many fields as you have available. If only one source attribute is available, you may map a single field.  The Amazon Ads destination will utilize all mapped fields for mapping purposes, yielding higher match rates if more fields are provided. For more information about the accepted identifiers, visit the [Amazon Ads hashed audience help page](https://advertising.amazon.com/dsp/help/ss/en/audiences#GA6BC9BW52YFXBNE).
+
+## Exported data / Validate data export {#exported-data}
+
+Once  your audience has been uploaded, you may validate your audience has been created and uploaded correctly using the following steps:
```
Comment (vfalon) [inline help/destinations/catalog/advertising/amazon-ads.md]: Extra space after Once.

**Review by vfalon — CHANGES_REQUESTED** [review]
Thanks for the updates, @vinvishy! The content looks very good now. Just a few minor comments to address before we continue. Thanks!

File: help/destinations/catalog/advertising/amazon-ads.md (hunk 8 above)
Comment (vinvishy) [reply help/destinations/catalog/advertising/amazon-ads.md]: Done.

File: help/destinations/catalog/advertising/amazon-ads.md (hunk 9 above)
Comment (vinvishy) [reply help/destinations/catalog/advertising/amazon-ads.md]: Done.

File: help/destinations/catalog/advertising/amazon-ads.md (hunk 10 above)
Comment (vinvishy) [reply help/destinations/catalog/advertising/amazon-ads.md]: Removed the extra blank lines.

File: help/destinations/catalog/advertising/amazon-ads.md (hunk 11 above)
Comment (vinvishy) [reply help/destinations/catalog/advertising/amazon-ads.md]: Removed the extra blank lines.

File: help/destinations/catalog/advertising/amazon-ads.md (hunk 12 above)
Comment (vinvishy) [reply help/destinations/catalog/advertising/amazon-ads.md]: Done.

File: help/destinations/catalog/advertising/amazon-ads.md (hunk 13 above)
Comment (vinvishy) [reply help/destinations/catalog/advertising/amazon-ads.md]: Done.

File: help/destinations/catalog/advertising/amazon-ads.md (hunk 17 above)
Comment (vinvishy) [reply help/destinations/catalog/advertising/amazon-ads.md]: Fixed.

File: help/destinations/catalog/advertising/amazon-ads.md (hunk 16 above)
Comment (vinvishy) [reply help/destinations/catalog/advertising/amazon-ads.md]: Done.

File: help/destinations/catalog/advertising/amazon-ads.md (hunk 15 above)
Comment (vinvishy) [reply help/destinations/catalog/advertising/amazon-ads.md]: Done.

File: help/destinations/catalog/advertising/amazon-ads.md (hunk 14 above)
Comment (vinvishy) [reply help/destinations/catalog/advertising/amazon-ads.md]: Done.

**Pull request merged**